cfpb/cfgov-refresh
# Add canonical URL link tag to Spanish site

Like #3744, but for the Spanish site.

## Additions

- adds canonical URL for SEO purposes to Spanish pages that use the `es-base.html` template
- adds canonical URL to Spanish Ask CFPB site template

## Removals

-

## Changes

-

## Testing

check out branch then:
1. `./runserver.sh`
1. go to http://localhost:8000/es/presentar-una-queja/ and view source and confirm that line 21 has the following element:
`<link rel="canonical" href="http://localhost:8000/es/presentar-una-queja/">`
and/or
1. go to  http://localhost:8000/ and view source and confirm that line 21 has the following element:
`<link rel="canonical" href="http://localhost:8000/es/">`
1. go to  http://localhost:8000/es/obtener-respuestas/categoria-enviar-dinero/ and view source and confirm that line 32 has the following element:
`<link rel="canonical" href="http://localhost:8000/es/obtener-respuestas/categoria-enviar-dinero/">`


## Screenshots

Spanish base template:
![screen shot 2018-02-01 at 4 09 17 pm](https://user-images.githubusercontent.com/702526/35703802-63989bca-076b-11e8-90ca-2c19921fd583.png)

Spanish Ask template:
![screen shot 2018-02-01 at 5 55 41 pm](https://user-images.githubusercontent.com/702526/35707802-35df185e-0779-11e8-8122-62d34debd762.png)


## Notes

-

## Todos

- @richaagarwal made the excellent suggestion that we take a look at combining some of the many base templates or portions of them to reduce duplicate code. So far, to add this 1 line of code, I'm duplicating it in 3 different base templates (the 2 in this PR plus 1 in #3768)

## Checklist
